custom/icds_reports/migrations/sql_templates/database_views/ccs_record_monthly_view.sql

```diff
@@ -49,7 +49,7 @@ CREATE VIEW ccs_record_monthly_view AS
         "ccs_record_monthly"."institutional_delivery_in_month" AS "institutional_delivery_in_month",
         "ccs_record_monthly"."is_ebf" AS "is_ebf",
         "ccs_record_monthly"."last_date_thr" AS "last_date_thr",
-        "ccs_record_monthly"."mobile_number" AS "mobile_number",
+        "awc_location_months"."contact_phone_number" AS "contact_phone_number",
```

Comment:
I think these are different things. One is the number of the mother (from the ccs record case) one is the contact number of the awc

Reply:
Oh, ok, so you would want the `phone_number` property from the `person_case` as Heena said: 

> We will not be able to use ccs_record property because the children being displayed in Child Beneficiary report can also be more than 3 years old

but as this property isn't in `person_cases_v2` UCR, wouldn't that mean that adding this property to the UCR wouldn't show phone number for the past records?

Reply:
The child beneficiary report will not use this view, so that problem doesn't really apply here. I think it is easiest to just use the property that is already here. When we add phone numbers to the rest of reports we can add it to the person UCR, but that is probably out of scope for the PR (and will delay implementation).

Reply:
Okay, thank you, I will revert the change.